<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-31VJ9UH" failures="1" tests="5" name="Test" time="0.105" errors="0" timestamp="2022-05-03T22:26:13 IST">
  <testcase classname="com.test.Demo" name="credentials1" time="0.015"/>
  <testcase classname="com.test.Demo" name="credentials2" time="0.003">
    <skipped/>
  </testcase> <!-- credentials2 -->
  <testcase classname="com.test.Demo" name="credentials2" time="0.002">
    <skipped/>
  </testcase> <!-- credentials2 -->
  <testcase classname="com.test.Demo" name="credentials2" time="0.003">
    <skipped/>
  </testcase> <!-- credentials2 -->
  <testcase classname="com.test.Demo" name="credentials2" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [214] but found [522]">
      <![CDATA[java.lang.AssertionError: expected [214] but found [522]
at com.test.Demo.credentials2(Demo.java:25)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- credentials2 -->
</testsuite> <!-- Test -->
